pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -3,8 +3,11 @@
 , glibmm, libsigcxx, lcms, boost, gettext, makeWrapper
 , gsl, python2, poppler, imagemagick, libwpg, librevenge
 , libvisio, libcdr, libexif, potrace, cmake, hicolor-icon-theme
+, python2Packages, scourSupport ? true
```

Comment:
we already pass in `python2`

Reply:
The python2Env includes packages that are only required at build time. Is it ok to include these packages at runtime even if they're not needed?

And I'll make the change and remove python2Packages.

Reply:
I'm relatively new to Nix. I really would like to use the existing python2Env but am unsure exactly how passing in `python2Env` and `makeWrapper` to `nativeBuildInputs` make `pythonPackages.scour` visible to `scour.inkscape.py`. I'm assuming `makeWrapper` wraps Inkscape in an executable shell script in which `PATH` and `PYTHONPATH` are set? Is make wrapper just called implicitly when added to `nativeBuildInputs`? I can't find documentation that elaborated on how all of this is achieved. Any suggestions would be much appreciated.

Reply:
scour needs to be available as an importable module. The `python2Env` provides a python executable that has access to the specified packages. Inkscape, at some point, records the reference to that Python interpreter, and thus will be able to import the given packages during runtime.

Reply:
You've got your hands full with all these PR's so extra thanks for taking the time to explain.